recipe/install_nvcc.sh

```diff
@@ -2,61 +2,66 @@
 
 set -xeuo pipefail
 
-# Default to using `nvcc` to specify `CUDA_HOME`.
-if [ -z ${CUDA_HOME+x} ]
-then
-    CUDA_HOME="$(dirname $(dirname $(which nvcc)))"
-fi
-
 # Set `CUDA_HOME` in an activation script.
 mkdir -p "${PREFIX}/etc/conda/activate.d"
-cat > "${PREFIX}/etc/conda/activate.d/${PKG_NAME}_activate.sh" <<EOF
+cat >"${PREFIX}/etc/conda/activate.d/${PKG_NAME}_activate.sh" <<EOF
 #!/bin/bash
 
-if [[ ! -d "${CUDA_HOME}" ]]
+# Default to using \$(cuda-gdb) to specify \$(CUDA_HOME).
+if [ -z \${CUDA_HOME+x} ]
+then
+    CUDA_HOME="\$(dirname \$(dirname \$(which cuda-gdb)))"
+fi
+
+if [[ ! -d "\${CUDA_HOME}" ]]
 then
-    echo "Cannot find: ${CUDA_HOME}"
+    echo "Cannot find: \${CUDA_HOME}"
     exit 1
 fi
-if [[ ! -f "${CUDA_HOME}/lib64/stubs/libcuda.so" ]]
+if [[ ! -f "\${CUDA_HOME}/lib64/stubs/libcuda.so" ]]
 then
-    echo "Cannot find: ${CUDA_HOME}/lib64/stubs/libcuda.so"
+    echo "Cannot find: \${CUDA_HOME}/lib64/stubs/libcuda.so"
     exit 1
 fi
-grep "CUDA Version ${PKG_VERSION}" ${CUDA_HOME}/version.txt &>/dev/null
-if [[ $? -ne 0 ]]
+grep "CUDA Version \${PKG_VERSION}" \${CUDA_HOME}/version.txt &>/dev/null
```

Comment:
```suggestion
grep "CUDA Version ${PKG_VERSION}" \${CUDA_HOME}/version.txt &>/dev/null
```

Reply:
Good catch. The other option here is to detect it at runtime since there's no longer anything version specific about this script / package.

Reply:
Other than the `cudatoolkit` run_exports